.github/workflows/automationTemplate.yaml

```diff
@@ -25,6 +25,7 @@ env:
   PYTHON_MIN_RELEASE_VERSION: ${{ vars.PYTHON_MIN_RELEASE_VERSION }}
   PYTHON_MAX_RELEASE_VERSION: ${{ vars.PYTHON_MAX_RELEASE_VERSION }}
   PYTHON_BLOCKED_VERSIONS_ARRAY: ${{ vars.PYTHON_BLOCKED_VERSIONS_ARRAY }}
+  ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN: ${{ secrets.ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN }}
```

Comment:
how the secrets are passed here?

Reply:
[This GitHub doc ](https://stackoverflow.com/questions/68008181/how-do-i-get-all-github-secrets-into-env-variables-for-actions-to-access-powers)can provide more details. Basically using `${{ secrets.ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN }}` will get the encrypted secret `ORYX_SDK_STORAGE_ACCOUNT_ACCESS_TOKEN`